src/RefreshButton.scss

```diff
@@ -0,0 +1,45 @@
+@import 'variables';
+
+.RefreshButton-svg {
+    width: 1.7rem;
+    height: 1.7rem;
+}
+
+button.btn.RefreshButton--disabled {
+    background-color: #FFF;
+    cursor: default;
+}
+
+button.btn.RefreshButton svg {
+    stroke: #000;
+}
+
+button.btn.RefreshButton--disabled svg {
```

Comment:
This rule is duplicated below. Please organize the rules in groups. Such as all rules for disabled state together. It will make it easier to spot duplication.